### PR TITLE
[RHACS] Added missing image registries paths and option values

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/ROX-25094

Changes:
- Added missing registry images paths
- Added missing scanner registry setting in Helm configuration parameters
- Created a new reference module because this info was used in two places.

Preview: 
- https://79099--ocpdocs-pr.netlify.app/openshift-acs/latest/release_notes/44-release-notes.html#image-versions_release-notes-44
- https://79099--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_ocp/install-secured-cluster-cloud-ocp.html#secured-cluster-services-config_install-secured-cluster-cloud-ocp
- https://79099--ocpdocs-pr.netlify.app/openshift-acs/latest/configuration/enable-offline-mode#image-versions_enable-offline-mode

### CODE REVIEW

File: modules/image-versions.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+You can manually pull, retag, and push {product-title} images to your registry. The current version includes the following images:
+
+.{product-title} images
+[%header,cols="1,1,2"]
+|===
+|Image |Description |Current version
+
+|Main
+|Includes Central, Sensor, Admission controller, and Compliance. Also includes `roxctl` for use in continuous integration (CI) systems.
```
Comment (snarayan-redhat) [inline modules/image-versions.adoc:18]:
```suggestion
|Includes Central, Sensor, Admission controller, and Compliance images. Also includes `roxctl` for use in continuous integration (CI) systems.
```

**Review by snarayan-redhat — COMMENTED** [review]
Looks great! Just one suggestion if it sounds technically correct.

File: modules/image-versions.adoc (hunk 1 above)
Comment (gaurav-nelson) [reply modules/image-versions.adoc:18]: Thank you @snarayan-redhat, good pick. However, these are note images, I've updated it to say `components` instead.

**Pull request merged**